eZ/Publish/API/Repository/Tests/Limitation/LanguageLimitationTest.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\API\Repository\Tests\Limitation;
+
+use eZ\Publish\API\Repository\Exceptions\UnauthorizedException;
+use eZ\Publish\API\Repository\Tests\BaseTest;
+use eZ\Publish\API\Repository\Values\User\Limitation\LanguageLimitation;
+
+class LanguageLimitationTest extends BaseTest
```

Comment:
I'm sorry, I've meant you should add test case to the file `eZ/Publish/API/Repository/Tests/Values/User/Limitation/LanguageLimitationTest.php`. Seems like namespace is wrong there and I didn't notice it when pasting it. The namespace needs to be fixed as well.